src/Renci.SshNet.Tests/Common/AsyncSocketListener.cs

```diff
@@ -193,12 +192,37 @@ private void ReadCallback(IAsyncResult ar)
             {
                 // Read data from the client socket.
                 bytesRead = handler.EndReceive(ar);
+
+                if (bytesRead > 0)
```

Comment:
I suppose this resolves an issue, but I always prefer to see the issue before a fix is applied for it.